src/main.rs

```diff
@@ -83,22 +89,41 @@ fn first_encoder(connector_info: &ConnectorInfo) -> Option<EncoderHandle> {
     connector_info.encoders().iter().next().copied()
 }
 
+const DEFAULT_CARD_PATH: &str = "/dev/dri/card0";
+
+#[derive(Parser)]
+struct Args {
+    /// Device file representing the GPU
+    #[arg(long, default_value = DEFAULT_CARD_PATH)]
+    card_path: String,
+
+    /// Connector to use, e.g. DP-1; if not provided some connected one will be selected
+    #[arg(long)]
+    connector: Option<String>,
+}
+
 fn main() {
+    let args = Args::parse();
+
     // TODO(bschwind) - Use libdrm to iterate over available DRM devices.
-    let gpu = Card::open("/dev/dri/card0");
+    let gpu = Card::open(&args.card_path);
     dbg!(gpu.get_driver().expect("Failed to get GPU driver info"));
     dbg!(gpu.get_bus_id().expect("Failed to get GPU bus ID"));
 
     let resources = gpu.resource_handles().expect("Failed to get GPU resource handles");
 
     print_connector_info(&gpu, &resources);
 
-    let force_probe = false;
-    let Some(first_connector_handle) = first_connected_connector(&gpu) else {
-        println!("No display connected, exiting");
+    let Some(first_connector_handle) = get_connected_connector(&gpu, &args.connector) else {
+        if let Some(name) = &args.connector {
+            println!("Display {name} does not exist or is not connected, exiting");
+        } else {
+            println!("No display connected, exiting");
```

Comment:
It may be confusing to call this a "Display" here while having a "connector" CLI argument. Let's align them. IMO "connector" is fine.

It would be even nicer UX to also mention on _which card_ the connector doesn't exist. Eg:

```suggestion
            println!("Connector {name} does not exist or is not connected to {}, exiting", &args.card_path);
        } else {
            println!("No display connected to {}, exiting", &args.card_path);
```